static/js/rest.js

```diff
@@ -29,11 +31,11 @@ export type RestState = {
   fetchStatus?: string,
 };
 
-const INITIAL_STATE: RestState = {
+const INITIAL_REST_STATE: RestState = {
```

Comment:
One thing I did in MM which you may want to do here is making the `RestState` type into a parameterized type, something like:

```js
type RestState<T> = {
  data: T,
  loaded: boolean,
  ...
};
```

makes Flow a little more useful when you're pulling the data out of the store later!